mod_reforged/hooks/skills/racial/serpent_racial.nut

```diff
@@ -1,4 +1,33 @@
 ::mods_hookExactClass("skills/racial/serpent_racial", function(o) {
+	local create = o.create;
+	o.create = function()
+	{
+		create();
+		this.m.Name = "Serpent";
+		this.m.Icon = "ui/orientation/serpent_orientation.png";
+		this.m.IsHidden = false;
+	}
+
+	o.getTooltip <- function()
+	{
+		local ret = this.skill.getTooltip();
+		ret.extend([
+			{
+				id = 12,
+				type = "text",
+                icon = "ui/icons/campfire.png",
+				text = "Reduces damage taken from fire by " + ::MSU.Text.colorRed("33%")
+			}/*,
+			{
+				id = 13,
```

Comment:
Why comment this out?

Reply:
This is a minor not very important change with hard to explain conditions. There are more minor effects scattered across the other racials.

I didn't want to go into that much detail for this first iteration of the racial tooltips.